pygluu/kubernetes/templates/helm/gluu/values.yaml

```diff
@@ -504,14 +504,14 @@ config:
     cnAwsAccessKeyId: ""
     # AWS Secret Access key that belong to a user/id with SecretsManagerReadWrite policy
     cnAwsSecretAccessKey: ""
-    # The URL of AWS secretsmanager service (if omitted, will use the one in specified region). Used only when global.configAdapterName and global.configSecretAdapter is set to aws.
+    # The URL of AWS secretsmanager service (if omitted, will use the one in the specified default region. Example: https://secretsmanager.us-west-1.amazonaws.com). Used only when global.configAdapterName and global.configSecretAdapter is set to aws.
     cnAwsSecretsEndpointUrl: ""
     # The prefix name of the secrets. Used only when global.configAdapterName and global.configSecretAdapter is set to aws.
     cnAwsSecretsNamePrefix: gluu
     # The default AWS Region to use, for example, `us-west-1` or `us-west-2`.
     cnAwsDefaultRegion: us-west-1
-    # The default profile to use.
-    cnAwsProfile: gluu
+    # The named profile to use. https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+    cnAwsProfile: default
```

Comment:
Do not change this. Also in the note put that this is a sensible default and is good to leave it as is . Note https://github.com/GluuFederation/cloud-native-edition/blob/4.5/pygluu/kubernetes/templates/helm/gluu/charts/config/templates/secrets.yaml#L129